client/src/components/RelatedProducts/ComparisonTable.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Item1 = styled.p`
+  text-align: left
+`;
+
+const Item2 = styled(Item1)`
+  text-align: right
+`;
+
+function Comparison() {
+  return (
+    <div>
```

Comment:
Is the nested div avoiding some error in react? Just wondering why the extra piece. No change needed though if it works.

Reply:
That was just something I was testing out, but forgot to remove.